bugwarrior/services/gitlab.py

```diff
@@ -432,7 +434,7 @@ def to_taskwarrior(self):
             if self.extra['type'] == 'todo' else self.record['title'])
         description = (
             self.record['body'] if self.extra['type'] == 'todo'
-            else self.record['description'])
+            else self.extra.get('description', ''))
```

Comment:
Is there a reason this can't just be `self.extra['description']`? Seems like the `description` key will always be set.

Reply:
I was getting keyerrors from pytest when using the simple accessor, maybe because some of the gitlab tests call to_taskwarrior directly and never go through the code that adds the extra (noticed this because those tests were failing on descriptions being None instead of '', hence the default)

Reply:
Hope you don't mind, I went ahead and made this change, adjusted the tests to accommodate it, rebased the branch to fix the unrelated CI failures, and pushed the amended commit to your branch.

Reply:
That's alright